resources/assets/less/livestreams.less

```diff
@@ -0,0 +1,191 @@
+.livestream-header {
+	background-size: cover;
+	background-repeat: no-repeat;
+	background-image: url("/images/headers/live-top.png");
+	height: 180px;
+	&__container {
+		display: flex;
+		justify-content: flex-start;
+		flex-wrap: wrap;
+		align-items: center;
+		height: 100%;
+		@media @desktop {
+			padding-left: 115px;
+			padding-right: 115px;
+		}
+		padding-left: 10px;
+		padding-right: 10px;
+	}
+	&__headline {
+		color: white;
+		margin: 0;
+		margin-bottom: 10px;
+	}
+	&__description {
+		color: white;
+	}
+}
+.livestream-page {
+	@media @desktop {
+		padding-left: 10% !important;
+		padding-right: 10% !important;
+	}
+	&__header {
+		color: @grey-5;
+		padding-bottom: 15px;
+	}
+	&__anchor-twitch,
+	&__anchor-twitch:link, 
+	&__anchor-twitch:visited, 
+	&__anchor-twitch:hover, 
+	&__anchor-twitch:active {
+		color: inherit;
+	    text-decoration: inherit;
+	    font-weight: inherit;
+	}
+}
+.livestream-featured {
+	padding-left: 10px;
+	padding-right: 10px;
+	&__info {
+		& * {
+			.default-text-shadow;
+			color: white;
+		}
+		position: absolute;
+		bottom: 10px;
+		left: 10px;
+	}
+	&__container {
+		transition: all 120ms ease-in-out;
+		float: none !important;
+		.default-box-shadow;
+		&:hover {
+			.thicker-box-shadow;
+		}
+		border-radius: 10px;
+		width: 100%;
+		padding-bottom: 75%;
+		background-size: cover;
+		background-repeat: no-repeat;
+		background-position: center center;
+	}
+	&:hover &__actions {
+		opacity: 1;
+	}
+	& &__actions {
+		color: white;
+		z-index: 9000;
+	}
+	&__actions {
+		@padding-offset: 9px;
+	    ._dektop-position {
+	      @media @desktop {
+	        justify-content: flex-start;
+	        width: 0;
+	        right: @padding-offset;
+	        bottom: 40px;
+
+	        opacity: 0;
+	        will-change: opacity;
+
+	        .forum-post:hover & {
+	          opacity: 1;
+	        }
+	      }
+	    }
+
+	    position: absolute;
+	    display: flex;
+	    flex-direction: column;
+	    align-items: center;
+	    justify-content: center;
+	    transition: opacity 120ms ease-in-out;
+
+	    right: 15px + @padding-offset;
+	    bottom: 15px;
+	    ._dektop-position;
+	}
+}
+.livestream-regular {
+	padding-left: 10px;
+	padding-right: 10px;
+	&:hover &__actions {
+		opacity: 1;
+	}
+	& &__actions {
+		color: white;
+		z-index: 9000;
+	}
+	word-break: break-all;

```

Comment:
that depends on definition of nicer. word-break works better here for livestream-regular blocks in my opinion.